src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -1531,6 +1585,98 @@ public void MediaType_SetThenGet_ValuesMatch(Uri remoteServer)
             Assert.Equal(MediaType, request.MediaType);
         }
 
+        [Theory, MemberData(nameof(CachableWebRequestParameters))]
+        public void GetResponseAsync_CachableParameters_GotSuccessfully(HttpWebRequestParameters requestParameters)
+        {
+            RemoteExecutor.Invoke(async (serializedParameters) =>
+            {
+                var parameters = JsonSerializer.Deserialize<HttpWebRequestParameters>(serializedParameters);
+                var options = new LoopbackServer.Options { UseSsl = false };
```

Comment:
`UseSsl = false` is the default, right?  Assuming yes, you should be able to delete this line and remove the `, options` from the call below.

Reply:
The idea was to clearly show that it checks only non-SSL scenario since I couldn't implement SSL-enabled test because it doesn't work without custom validation delegate and, when that delegate is set, caching gets disabled. But, I removed this line per your request.